node/res/eden.json

```diff
@@ -3,7 +3,11 @@
     "id": "para_eden",
     "chainType": "Live",
     "bootNodes": [
-        "/dns4/paraboot1.nodleprotocol.io/tcp/30333/p2p/12D3KooWMy24nxEfm92nmmzkSw28Ek6xcv8EkBhbCuWviU6cX3F8"
+		"/dns4/node-6904336564467159040-0.p2p.onfinality.io/tcp/14142/p2p/12D3KooWCcoD6uqiae1pA4LZHd5Zm1xGhN4RgBAAt9EUkrvhjJan",
+		"/dns4/node-6904339013521264640-0.p2p.onfinality.io/tcp/19584/p2p/12D3KooWCV2p3pHf8UmDUX9J8bTAKUCqyqZcmByUrzYh2LRoiVQH",
+		"/dns4/node-6904339159696953344-0.p2p.onfinality.io/tcp/14068/p2p/12D3KooWLthb2JHr5GPSG88cnkTfJzsQmTZ2HsiN4Fu8yMNzuGvv",
+		"/dns4/node-6904339635149418496-0.p2p.onfinality.io/tcp/20539/p2p/12D3KooWD2KoGVv6u5y27jz9MSvrcK1E6dYUS2JpyGkLf3zkXtre",
+		"/dns4/node-6904339780776906752-0.p2p.onfinality.io/tcp/14896/p2p/12D3KooWD61umWiWrA1cP3wxWcKAcdPyf7kFMRQxJteWe8SSiU6T"
```

Comment:
@rajesh-nodle can you update those to match the most recently deployed nodes on OF?